internal/arduino/sketch/testdata/profiles/profile_with_libraries.yml

```
@@ -0,0 +1,13 @@
+profiles:
+  giga:
+    fqbn: arduino:mbed_giga:giga
+    platforms:
+      - platform: arduino:mbed_giga (4.3.1)
+    libraries:
+      - FlashStorage (1.2.3)
+      - dir: /path/to/system/lib
+      - dir: path/to/sketch/lib
+      - git: https://github.com/username/HelloWorld.git#v2.13
```

Why `#`? I think it is more common `@` in this case
```suggestion
      - git: https://github.com/username/HelloWorld.git@v2.13
```